CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
 cmake_minimum_required(VERSION 3.6 FATAL_ERROR)
+
+option(USE_VCPKG "assume vcpkg presence and include it so find_package finds it." FALSE)
+if(USE_VCPKG)
+  include(${VCPKG_DIR})
```

Comment:
Is this a pattern the vcpkg devs recommend? I believe you could achieve the same thing with the CMake [CMAKE_PROJECT_TOP_LEVEL_INCLUDES](https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_TOP_LEVEL_INCLUDES.html) feature.

Reply:
Oh ! We didn't know about that feature. Many thanks for the tips !
I will close this pull request.